### PR TITLE
setup for TSQL Grammar Simple `SELECT`

### PR DESCRIPTION
**Pull request by sundarshankar89**

### CODE REVIEW

**Review by vil1 — CHANGES_REQUESTED** [review]

File: core/src/test/scala/com/databricks/labs/remorph/parsers/tsql/ParserTestCommon.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.databricks.labs.remorph.parsers.tsql
+
+import org.scalatest.Assertions
+
+import com.databricks.labs.remorph.parsers.intermediate.{NamedTable, Relation, TreeNode}
+import org.antlr.v4.runtime.{CharStreams, CommonTokenStream, RuleContext}
+import org.scalatest.{Assertion, Assertions}
+
+// [TODO]: Need to consolidate for all the dialects.
```
[the remark this answers is not in the document]
Comment (sundarshankar89) [reply core/src/test/scala/com/databricks/labs/remorph/parsers/tsql/ParserTestCommon.scala:9]: @vil1 do you want me to make the changes or you will apply I rebase happy to do both.

[the remark this answers is not in the document]
Comment (vil1) [reply core/src/test/scala/com/databricks/labs/remorph/parsers/tsql/ParserTestCommon.scala:9]: Let's do the change in this PR. I can push a commit on this branch of you want.

[the remark this answers is not in the document]
Comment (sundarshankar89) [reply core/src/test/scala/com/databricks/labs/remorph/parsers/tsql/ParserTestCommon.scala:9]: sure go ahead.

[the remark this answers is not in the document]
Comment (vil1) [reply core/src/test/scala/com/databricks/labs/remorph/parsers/tsql/ParserTestCommon.scala:9]: done

File: core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlAstBuilder.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  override def visitSelect_statement_standalone(ctx: Select_statement_standaloneContext): ir.TreeNode = {
+    val rawExpression = ctx.select_statement().query_expression().query_specification()
+    val column_expression = rawExpression
+      .select_list()
+      .select_list_elem()
+      .asScala
+      .map(_.accept(new TSqlExpressionBuilder))
+    val from_table = rawExpression.table_sources().accept(new TSqlRelationBuilder)
+
+    ir.Project(from_table, column_expression)
```
Comment (vil1) [inline core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlAstBuilder.scala:32]:
```suggestion
    val columnExpressions = rawExpression
      .select_list()
      .select_list_elem()
      .asScala
      .map(_.accept(new TSqlExpressionBuilder))
    val fromTable = rawExpression.table_sources().accept(new TSqlRelationBuilder)

    ir.Project(fromTable, columnExpressions)
```

**Review by vil1 — APPROVED** [review]
lgtm

**Pull request merged**